scripts/generate_query_config.pl

```diff
@@ -64,9 +64,6 @@
     if ($line =~ /^(\/\/)?\s*#\s*define\s+(MBEDTLS_\w+).*/) {
         my $name = $2;
 
-        # Skip over the macro that prevents multiple inclusion
-        next if "MBEDTLS_CONFIG_H" eq $name;
```

Comment:
I've removed the include guard from `config.h` / `mbedtls_config_defines.h`, (since the guard against multiple inclusion is now in `mbedtls_config.h`.